_articles/discussion.md

```diff
@@ -11,56 +11,42 @@ category: collaboration-and-communication
 }
 </style>
 
-`Discussion` forums are a way to easily communicate with others, similar to a message board, which is visible to all those with whom a project is shared. In this guide you will learn how to:
+`Discussion` forums are a space to communicate with others, similar to a message board. The forum is visible to users who have access to the `Project`. 
+<br>
+### Creating a New Message Thread
 
-* Create a new message thread
-* Respond to a current thread
-* Manage notifications
-* Moderate a discussion forum
+The Discussion tab can be found in the Project navigation tabs on the Project main page. Anyone with Project access may view discussion threads contained in the Project, create new threads, or reply to existing threads.
 
-## Creating a New Message Thread
+<img style="width: 100%;" src="../assets/images/Discussion_Main.png">
 
-The `Discussion` tab can be found in the project navigation tabs on your project's main page. Anyone with `Project` access may view discussion threads contained in the project, create new threads, or reply to existing threads.
+To create a new thread, click the **New Thread** button at the top of the Discussion tab.
 
-<img style="width: 65%;" src="../assets/images/Discussion Main.png">
+<img style="width: 100%" src="../assets/images/NewThread_Dialogue.png">
```

Comment:
This image is broken when I try to run this locally. May want to double check that it's working.

Reply:
oops that is because it wasn't committed! That should be fixed now.